src/main/java/com/launchacademy/reviews/controllers/PizzaStylesApiV1Controller.java

```diff
@@ -63,4 +66,21 @@ public Object getById(@PathVariable Integer id){
       return customError.doesntExists();
     }
   }
+
+  @GetMapping("/{id}/{sortOption}")
+  public Object getByIdAndSortReviews(@PathVariable Integer id, @PathVariable String sortOption){
+    Map<String, PizzaStyle> map = new HashMap<>();
+    if(pizzaStyleService.findById(id).isPresent()){
+      PizzaStyle pizzaStyle = pizzaStyleService.findById(id).get();
+      if (sortOption.equals("RatingDesc")) {
+        pizzaStyle.setReviews(reviewService.findByPizzaStyleIdOrderByRatingDesc(id));
+      } else if (sortOption.equals("RatingAsc")) {
+        pizzaStyle.setReviews(reviewService.findByPizzaStyleIdOrderByRatingAsc(id));
+      }
```

Comment:
Great job taking advantage of the named queries! If this was to continue growing and allow for different sorting options, it may be worth looking into how to pass a `Sort` parameter to make this more dynamic and avoid a long if/elseIf chain.  Nothin you need to implement now.

https://www.baeldung.com/spring-data-sorting

Reply:
I am not sure how to remove the if-else since the query depends on the `sortOption` value passed from the fetch request

Reply:
Great job and you will always need one if else statement to handle if the query should be ascending/descending, but past that we could restructure the `sortOption` to include both the `field` to be sorted by and the `order` to do it.  Than you can always use `Sort.by(sortOptions.field)` instead of having to hardcode the string `Sort.by("rating")`.  I'm going to go ahead and merge this so, it's not blocking, but it may be worth another refactor story later on this week.